themes/conventional-commits/layouts/_default/single.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html class="{{ if .Site.Language.Params.rtl }}rtl{{ end }}">
```

Comment:
I think you forgot to handle the top banner section as well:
<img width="1195" alt="Screenshot 2024-10-22 at 13 37 42" src="https://github.com/user-attachments/assets/59d7de85-057a-4153-8886-d0e2488e9977">

The text overlaps the background image

Reply:
According to the Arabic language layout, the title should be on the right, so I made the image stick to the left if the language was Arabic.

Reply:
Looks perfect now, thank you @NAMEER242 !
<img width="1008" alt="Screenshot 2024-10-23 at 10 56 54" src="https://github.com/user-attachments/assets/f558415d-d625-46da-a680-949008ddc45b">